docs/mp/lra.adoc

```diff
@@ -0,0 +1,460 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021, 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= LRA
+:toc:
+:toc-placement: preamble
+:description: Long Running Actions
+:keywords: helidon, mp, lra
+:h1Prefix: MP
+:feature-name: Long Running Actions
+:spec-version: 1.0-RC3
+:spec-name: Micro Profile {feature-name} specification
+:javadoc-link: https://download.eclipse.org/microprofile/microprofile-lra-{spec-version}/apidocs/org/eclipse/microprofile/lra/annotation/
+:rootdir: {docdir}/..
+
+== Contents
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+- <<Reference, Reference>>
+
+== Overview
+
+Distributed transactions for microservices are known as SAGA design patterns and are defined by the {microprofile-lra-spec-url}[{spec-name}].
+Unlike well known XA protocol, LRA is asynchronous and therefore much more scalable. Every LRA JAX-RS resource (<<Participant, participant>>) defines endpoints to be invoked when transaction needs to be _completed_ or _compensated_.
+
+
+include::{rootdir}/includes/dependencies.adoc[]
+[source,xml]
+----
+<dependency>
+  <groupId>io.helidon.microprofile.lra</groupId>
+  <artifactId>helidon-microprofile-lra</artifactId>
+</dependency>
+<!-- Support for Narayana coordinator -->
+<dependency>
+  <groupId>io.helidon.lra</groupId>
+  <artifactId>helidon-lra-coordinator-narayana-client</artifactId>
+</dependency>
+----
+
+== Usage
+
+LRA transactions need to be coordinated over REST API by the LRA coordinator. <<Coordinator, Coordinator>>
+keeps track of all transactions and calls the `@Compensate` or `@Complete` endpoints for all participants involved in the particular
+transaction. LRA transaction is first started, then joined by <<Participant, participant>>.
+Participant reports the successful finish of transaction by calling complete. Coordinator then calls the JAX-RS
```

Comment:
The participant reports the successful finish of the transaction by calling it complete. The coordinator

Reply:
Fixed)